frontend/src/app/settings/core/tools-settings/tool.service.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable, tap } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+export type CreateTool = {
+  name: string;
+};
+
+export type Tool = CreateTool & {
+  id: number;
+};
+
+export type ToolVersion = {
+  id: number;
+  name: string;
+  is_recommended: boolean;
+  is_deprecated: boolean;
```

Comment:
I would make this consistent with the `PatchToolVersion` (use `isRecommended` and `isDeprecated`)

Reply:
We'd need to map all responses then (because the backend returns it in the snake case format). What do you think about it?

Reply:
Now I see the problem with this one. Yes, then I think it's the best solution to just keep it in the snake case so as not to include too much overhead with the need for mapping responses.